core/src/package.rs

```diff
@@ -45,6 +50,62 @@ pub enum PackageImplementation {
     Native,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub enum ArgType {
+    #[serde(alias = "string")]
+    String,
+    #[serde(alias = "int", alias = "integer", alias = "i64")]
+    Integer,
+    #[serde(
+        rename = "Unsigned integer",
```

Comment:
Or maybe you can ignore this if "unsigned integer" already is autoderived based on the name on the enum variant.

Reply:
Nope, this is intentional. The serialised name (that is, when going from `ArgType` to JSON, it will name it according to the enum, (that is `String`, `Integer`, `Float`), and this rename changes `UnsignedInteger` to be `Unsigned integer`. This is the value sent to the front-end, that is displayed in the Playground. So it isn't inconsistent with the other enums since they also serialise with uppercase first character. Of course we can change this "convention" to use lowercase names (then we would have to rename the others as well). Let me know if you think we should do this (I have no problem changing everything to lowercase if we feel that it is better).

The aliases, however, is only used for deserialisation (that is, what the developers may put in the package manifest). They can either put the enum name (`String`, `Integer`, `Unsigned integer`, `Float`) or any of the specific aliases (which are lowercase). Most likely, people will use the aliases, so while a manifest may look like `{name: "Size", type: "uint"}`, when serialising the package info to send it to the front-end, it will have a more descriptive name `{name: "Size", type: "Unsigned integer"}`. I did this so that package developers can specify their type with a name that possibly more closely represents the name that it has in their programming language, while a more friendly name will be used when serialising.

Of course, we can change this in many ways. We can either just use `serde(rename)` and enforce one name for each enum, which will be the same for both deserialising (from the manifest) and serialising (to send to the front-end). Or, we can keep it as-is, but rename the serialisation name to use lowercase only. I added some aliases to make it a bit easier for developers, but we can be more strict if you think that is better! Feel free to tell me your thoughts, I am happy to change it if you think we should!